docs/get-started/sveltekit.md

```diff
@@ -0,0 +1,342 @@
+
+Run the following command to install the addon.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+   'svelte/svelte-csf-addon-install.yarn.js.mdx',
+   'svelte/svelte-csf-addon-install.npm.js.mdx',
+   'svelte/svelte-csf-addon-install.pnpm.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<Callout variant="info">
+
+The community actively maintains the Svelte CSF addon but still lacks some features currently available in the official Storybook Svelte framework support. For more information, see [the addon's documentation](https://github.com/storybookjs/addon-svelte-csf).
+
+</Callout>
+
+## API
+
+### Parameters
+
+This framework contributes the following [parameters](../writing-stories/parameters.md) to Storybook, under the `sveltekit_experimental` namespace:
+
+#### `forms`
+
+Type: `{ enhance: TK }`
+
+Provides mocks for the `$app/forms` module.
+
+##### `forms.enhance`
+
+Type: TK
+
+A callback that will be called when a form with `use:enhance` is submitted.
+
+#### `hrefs`
+
+Type: `Record<[path: string], (to: string, event: TK) => void | { callback: (to: string, event: TK) => void, asRegex?: boolean }>`
+
+If you have an `<a />` tag inside your code with the `href` attribute that matches one or more of the links defined (treated as regex based if the `asRegex` property is `true`) the corresponding `callback` will be called. If no matching `hrefs` are defined, an action will be logged to the [Actions panel](../essentials/actions.md). See [Mocking links](#mocking-links) for an example.
+
+#### `navigation`
+
+Type: TK
+
+Provides mocks for the `$app/navigation` module.
+
+##### `navigation.goto`
+
+Type: TK
+
+A callback that will be called whenever `goto` is called. If no function is provided, an action will be logged to the [Actions panel](../essentials/actions.md).
+
+##### `navigation.pushState`
+
+Type: TK
+
+A callback that will be called whenever `pushState` is called. If no function is provided, an action will be logged to the [Actions panel](../essentials/actions.md).
+
+##### `navigation.replaceState`
+
+Type: TK
+
+A callback that will be called whenever `replaceState` is called. If no function is provided, an action will be logged to the [Actions panel](../essentials/actions.md).
+
+##### `navigation.invalidate`
+
+Type: TK
+
+A callback that will be called whenever `invalidate` is called. If no function is provided, an action will be logged to the [Actions panel](../essentials/actions.md).
+
+##### `navigation.invalidateAll`
+
+Type: TK
+
+A callback that will be called whenever `invalidateAll` is called. If no function is provided, an action will be logged to the [Actions panel](../essentials/actions.md).
+
+##### `navigation.afterNavigate`
+
+Type: TK
+
+An object that will be passed to the `afterNavigate` function, which will be invoked when the `onMount` event fires.
+
+#### `stores`
```

Comment:
Similar to above, maybe linking to SvelteKit's docs on the types is best?

https://kit.svelte.dev/docs/modules#$app-stores